lsp-fsharp.el

```diff
@@ -217,8 +217,7 @@ disable if `--backgorund-service-enabled' is not used"
 
 (defun lsp-fsharp--make-init-options ()
   "Init options for F#."
-  `((:AutomaticWorkspaceInit . t))
```

Comment:
If there is such setting on the server we should map it to emacs defcustom. (I am assuming that someone might want to change it).

Reply:
I'm happy to implement that; given @TOTBWF's comments about the issues it can cause, it should probably be marked risky and documented clearly.